### PR TITLE
fix: Improved SEO score

### PR DESCRIPTION
**Pull request by parasss19**
## Description
 - Added proper meta tags to improve the SEO score
 - Also properly added keywords , titles, description etc to improve SEO

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -19,6 +19,31 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
+      <head>
+      <title>ProjectHut-Learn.Build.Share</title>
+        <meta name="title" content="ProjectHut-Learn.Build.Share" />
+        <meta
+          name="description"
+          content="ProjectsHut is an open source web app that enables users to freely publish their projects and create user profiles within the platform "
+        />
```
Comment (priyankarpal) [inline app/layout.tsx:28]: In the `<head>` section, there are two conflicting title & description elements. One is directly in the `<head>` and the other is in the `<meta>` tags. Remove the redundant title in the `<head>` section.

```suggestion
     <meta name="title" content={metadata.title} />
        <meta name="description" content={metadata.description} />
```

File: app/layout.tsx [hunk 2]
```diff
@@ -19,6 +19,31 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
+      <head>
+      <title>ProjectHut-Learn.Build.Share</title>
```
Comment (priyankarpal) [inline app/layout.tsx:23]: remove the `title` 
```suggestion

```

File: app/layout.tsx [hunk 3]
```diff
@@ -19,6 +19,31 @@ export default function RootLayout({
...
+        />
+        <meta name="language" content="en" />
+        <meta name="keywords" content="ProjectHut, projects, educational resources, wide range of projects, high-quality projects, React, Nodejs , javascript, open source, contribution , learners" />
+        <meta name="author" content="Priyankar Pal" />
+        <meta name="robots" content="index, follow" />
+        <meta name="revisit-after" content="7 days" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        {/* Open Graph meta tags for website preview */}
+        <meta property="og:title" content="ProjectHut - Learn.Build.Share" />
+        <meta property="og:description" content="ProjectsHut is an open source web app that enables users to freely publish their projects and create user profiles within the platform." />
```
Comment (priyankarpal) [inline app/layout.tsx:37]:
```suggestion
        <meta property="og:title" content={metadata.title} />
        <meta property="og:description" content={metadata.description} />
```

File: app/layout.tsx [hunk 4]
```diff
@@ -19,6 +19,31 @@ export default function RootLayout({
...
+        <meta property="og:url" content="https://projectshut.vercel.app/" /> {/* Replace with the actual URL of the page */}
```
Comment (priyankarpal) [inline app/layout.tsx:38]:
```suggestion
        <meta property="og:url" content="https://projectshut.vercel.app/" /> 
```

File: app/layout.tsx [hunk 5]
```diff
@@ -19,6 +19,31 @@ export default function RootLayout({
...
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content="ProjectHut" />
+        {/* Twitter meta tags for website preview */}
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta property="twitter:title" content="ProjectHut - Learn.Build.Share" />
+        <meta property="twitter:description" content="ProjectsHut is an open source web app that enables users to freely publish their projects and create user profiles within the platform." />
```
Comment (priyankarpal) [inline app/layout.tsx:44]:
```suggestion
         <meta property="twitter:title" content={metadata.title} />
        <meta property="twitter:description" content={metadata.description} />
```

File: app/layout.tsx [hunk 6]
```diff
@@ -19,6 +19,31 @@ export default function RootLayout({
...
+        <meta name="revisit-after" content="7 days" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        {/* Open Graph meta tags for website preview */}
+        <meta property="og:title" content="ProjectHut - Learn.Build.Share" />
+        <meta property="og:description" content="ProjectsHut is an open source web app that enables users to freely publish their projects and create user profiles within the platform." />
+        <meta property="og:url" content="https://projectshut.vercel.app/" /> {/* Replace with the actual URL of the page */}
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content="ProjectHut" />
+        {/* Twitter meta tags for website preview */}
+        <meta name="twitter:card" content="summary_large_image" />
```
Comment (priyankarpal) [inline app/layout.tsx:42]: can you explain me about `"summary_large_image"`?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: app/layout.tsx (hunk 6 above)
Comment (parasss19) [reply app/layout.tsx:42]: - Summary cards are Twitter's way of presenting a link with a title, description, and a featured image to make it more visually appealing and informative. 
-  You can also add the preview image you want to display

File: app/layout.tsx (hunk 1 above)
Comment (parasss19) [reply app/layout.tsx:28]: Done

**Comment by parasss19** [comment]
Hey @priyankarpal I resolved all the issues Please review it now

**Pull request closed without merging**